java/java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -3972,7 +3977,7 @@ else if (Kind.VARIABLE == tree.getKind()) {
         }
         return result;
     }
-
+    
```

Comment:
Hi can you remove this space added change

Reply:
Indeed..

Reply:
comments addressed